packages/client/src/gen/coordinator/index.ts

```diff
@@ -2525,7 +2493,7 @@ export interface ReactionResponse {
    * @type {{ [key: string]: any; }}
    * @memberof ReactionResponse
    */
-  custom?: { [key: string]: any };
+  custom: { [key: string]: any };
```

Comment:
```suggestion
  custom?: { [key: string]: any };
```

The newest schema marks this field as optional.